bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -366,8 +366,6 @@ def _validate_chunk_row_in_progress(self, chunk):
         """Helper for :meth:`_validate_chunk`"""
         assert self.state == self.ROW_IN_PROGRESS
         self._validate_chunk_status(chunk)
-        if not chunk.HasField('commit_row') and not chunk.reset_row:
-            _raise_if(not chunk.timestamp_micros or not chunk.value)
```

Comment:
> setting timestamp_micros = 0 is the same as not setting it at all

Right:  the question is, is it valid to be start a new cell without a timestamp? (we are in the "row-in-progress" state, which means we have no active / partial cell).  I'm fine to defer to @garye if the answer is "yes, that is valid."

It'd be breat to have public docs for the chunking stuff, too. ;)